### DISCUSSION TITLE
Welcome to Acumatica-Surveys Discussions!

### DISCUSSION THREAD

**Discussion by MvF-Evangelista** [Acumatica/Acumatica-Surveys#42] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our Acumatica Developer community. 
  
  We hope that you:
  * Ask questions 
  * Share ideas and feature requests
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community where we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us how you would like to contribute to Acumatica Surveys.

**Comment by FutureKenya** [comment]
Hi,
How can I get this latest Acumatica Survey package that support survey for external users
